docs/code-snippets/excel-snippets.yaml

```
@@ -1,8 +1,76 @@
+'ExcelScript.CommentRichContent#richContent:member':
+ - |-
+    /**
+     * This sample finds overdue work items in a table and 
+     * lets their owners know with a comment that uses an @mention.
+     * 
+     * This assumes the worksheet has a table with the columns: 
+     * "Work Item", "Project", "Owner", "Due Date"
+     */
+    function main(workbook: ExcelScript.Workbook) {
+      let currentSheet = workbook.getActiveWorksheet();
+
+      // Get the "Owner" column range and values.
+      let table = currentSheet.getTables()[0];
+      let ownerColumnRange = table.getColumn("Owner").getRangeBetweenHeaderAndTotal();
+      let ownerColumnValues = ownerColumnRange.getValues();
+
+      // Get the "Due Date" column range and values.
+      let dueDateColumnRange = table.getColumn("Due Date").getRangeBetweenHeaderAndTotal();
+      let dueDateColumnValues = dueDateColumnRange.getValues();
+
+      // Look for overdue work items.
+      for (let row = 0; row < dueDateColumnValues.length; row++) {
+
+        // Convert the Excel date into a JavaScript date. 
+        // This is necessary because the two platforms store
+        // their dates as different numerical values.
+        let dueDate = new Date(Math.round((dueDateColumnValues[row][0] - 25569) * 86400 * 1000));
+
+        // Check if the current date is greater than the due date.
+        if (dueDate.valueOf() < Date.now()) {
```

I know my suggestion is functionally the same but I think it would help the user if the comment and the code matched. Could instead update the comment.
```suggestion
        if (Date.now() > dueDate.valueOf()) {
```